Misc/NEWS.d/next/Library/2018-06-20-09-12-21.bpo-23224.zxCQ13.rst

```diff
@@ -0,0 +1,6 @@
+Fix segfaults when creating :class:`lzma.LZMADecompressor` and
+:class:`bz2.BZ2Decompressor` objects without calling ``__init__()``, and fix
```

Comment:
Segfaults apply for `*Compressor` objects too.

Reply:
@izbyshev I don't think so. Please explain why you think segfaults can occur with `*Compressor` objects.

Reply:
An uninitialized `*Compressor` object will crash if any of its methods is called:
```
from bz2 import BZ2Compressor as C
C.__new__(C).compress(b'')
Segmentation fault (core dumped)
```

Reply:
Post this PR:

```python
Python 3.12.0a5+ (heads/bpo-23224-lzma-bz2-segfault-dirty:66dc4eb5f9, Feb 23 2023, 00:13:14) [Clang 14.0.0 (clang-1400.0.29.202)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from bz2 import BZ2Compressor as C
>>> C.__new__(C).compress(b'')
b''
```

Marking this as resolved.